.github/workflows/ci-erd-openapi.yml

```diff
@@ -1,12 +1,12 @@
-# Update OpenAPI docs so that they remain up to date with the application
-name: Update OpenAPI Docs
+# Update database ERD diagrams and OpenAPI docs so that they remain up to date with the application
+name: Update Database ERD Diagrams and OpenAPI Docs
 
 on:
   pull_request:
     paths:
       - api/**
       - Makefile
-      - .github/workflows/ci-openapi.yml
+      - .github/workflows/ci-erd-openapi.yml
```

Comment:
Can you keep the filename as is 🙏🏽 it'll prevent merge conflicts if we end up wanting to pull in changes from the upstream that this file came from

Reply:
makes sense. done